2021/24xxx/CVE-2021-24162.json

```diff
@@ -1,18 +1,92 @@
 {
-    "data_type": "CVE",
-    "data_format": "MITRE",
-    "data_version": "4.0",
-    "CVE_data_meta": {
-        "ID": "CVE-2021-24162",
-        "ASSIGNER": "cve@mitre.org",
-        "STATE": "RESERVED"
-    },
-    "description": {
-        "description_data": [
-            {
-                "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
-            }
+  "CVE_data_meta": {
+    "ID": "CVE-2021-24162",
+    "ASSIGNER": "contact@wpscan.com",
+    "STATE": "PUBLIC",
+    "TITLE": "Responsive Menu < 4.0.4 - CSRF to Settings Update"
+  },
+  "data_format": "MITRE",
+  "data_type": "CVE",
+  "data_version": "4.0",
+  "generator": "WPScan CVE Generator",
+  "affects": {
+    "vendor": {
+      "vendor_data": [
+        {
+          "vendor_name": "TODO",
+          "product": {
+            "product_data": [
+              {
+                "product_name": "Responsive Menu – Create Mobile-Friendly Menu",
+                "version": {
+                  "version_data": [
+                    {
+                      "version_affected": "<",
+                      "version_name": "4.0.4",
+                      "version_value": "4.0.4"
+                    }
+                  ]
+                }
+              },
+              {
+                "product_name": "",
+                "version": {
+                  "version_data": [
+                    {
+                      "version_affected": "<",
+                      "version_name": "4.0.4",
+                      "version_value": "4.0.4"
+                    }
+                  ]
+                }
+              }
+            ]
+          }
+        }
+      ]
+    }
+  },
+  "description": {
+    "description_data": [
+      {
+        "lang": "eng",
+        "value": "\"Attackers could craft a request and trick an administrator into importing all new settings. These settings could be modified to include malicious JavaScript, therefore allowing an attacker to inject payloads that could aid in further infection of the site.\""
```

Comment:
The description must, at least, have the product name, and “\” … “\” characters should be removed.